frontend/device/remarkable/device.lua

```diff
@@ -112,12 +113,30 @@ end
 
 
 function Remarkable:init()
+    -- Check if we were launched from Oxide launcher
+    --- @fixme this could be changed to check parent process of KOReader instead
+    local oxide_running = os.execute("systemctl is-active --quiet tarnish") == 0
+    logger.info(string.format("Oxide running?: %s", oxide_running))
+
+    -- experiment
+    -- logger.info("PPID:")
+    -- local parent_process = os.execute("echo $PPID")
+    -- os.execute("ps | grep $PPID")
+    -- logger.info(string.format("parent proccess is oxide?: %s", parent_process_is_oxide))
+
     self.screen = require("ffi/framebuffer_mxcfb"):new{device = self, debug = logger.dbg}
     self.powerd = require("device/remarkable/powerd"):new{
         device = self,
         capacity_file = self.battery_path,
         status_file = self.status_path,
     }
+
+    event_map = require("device/remarkable/event_map")
+    -- If we were launched from Oxide, remove Power from the event map
+    if oxide_running then
+        event_map[116] = "noop"
```

Comment:
`= nil`?

Reply:
I was following [this comment](https://github.com/Eeems-Org/oxide/issues/135#issuecomment-811633258). Happy to change if it works.

Reply:
I'm fairly sure it will, and will make more sense than this, as we don't actually have a `noop` key event ^p^.

Reply:
Gotcha. So setting to `nil` is kinda like unsetting that value in the hash.

Reply:
```suggestion
        event_map[116] = nil
```